Makefile

```diff
@@ -134,16 +134,26 @@ test-e2e:
 	go test ./test/e2e/ -v -ginkgo.v
 
 
+# MARKDOWNLINT_IMAGE = docker.io/peterdavehello/markdownlint
```

Comment:
can you please remove this comment as well?

Reply:
done